### PR TITLE
[security] Upgrade kernel from 3.16.57-2 to 3.16.68-2 on 201803 branch

### PR DESCRIPTION
**Pull request by qiluo-msft**
TODO:
- [x] Recompile opennsl-modules
- [ ] Recompile Nephos's nps-modules and update platform/nephos/sdk.mk
- [x] Update Mellanox SAI/SDK/FW download links

Tested:
- [x] Broadcom image on Dell 6100
- [x] Mellanox image on Mellanox 2700

### CODE REVIEW

File: platform/broadcom/sdk.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # mock link here, need to be replaced by real link from MSFT
 
-BRCM_OPENNSL_KERNEL = opennsl-modules-3.16.0-6-amd64_3.4.1.11-7_amd64.deb
-$(BRCM_OPENNSL_KERNEL)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/opennsl-modules-3.16.0-6-amd64_3.4.1.11-7_amd64.deb?sv=2015-04-05&sr=b&sig=HGePoJSCcURIMW3bPRh5iXlx6z5SWiElmqD44mqUchI%3D&se=2155-08-28T16%3A31%3A48Z&sp=r"
+BRCM_OPENNSL_KERNEL = opennsl-modules-3.16.0-9-amd64_3.4.1.11-7_amd64.deb
+$(BRCM_OPENNSL_KERNEL)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/opennsl-modules-3.16.0-9-amd64_3.4.1.11-7_amd64.deb"
```
Comment (lguohan) [inline platform/broadcom/sdk.mk]: this needs to be recompiled.

File: platform/nephos/sdk.mk [hunk 2]
```diff
@@ -1,5 +1,5 @@
 SDK_VERSION = 2.0.5
-LINUX_VER = 3.16.0-6
+LINUX_VER = 3.16.0-9
 SDK_COMMIT_ID = f2e56f
 NEPHOS_NPS_KERNEL = nps-modules-$(LINUX_VER)_$(SDK_VERSION)_$(SDK_COMMIT_ID)_amd64.deb
 $(NEPHOS_NPS_KERNEL)_URL = "https://github.com/NephosInc/SONiC/raw/master/sdk/nps-modules-$(LINUX_VER)_$(SDK_VERSION)_$(SDK_COMMIT_ID)_amd64.deb"
```
Comment (qiluo-msft) [inline platform/nephos/sdk.mk]: >$(NEPHOS_NPS_KERNEL)_URL [](start = 0, length = 24)

Need Nephos team recompile the kernel module and update the download URL.

**Comment by qiluo-msft** [comment]
Test Broadcom build by a Jenkins job
[consoleText.txt](https://github.com/Azure/sonic-buildimage/files/3316583/consoleText.txt)

**Comment by qiluo-msft** [comment]
Test Mellanox build by a Jenkins job
[consoleText.txt](https://github.com/Azure/sonic-buildimage/files/3317267/consoleText.txt)

**Pull request merged**